tasks.py

```diff
@@ -86,27 +86,53 @@ def createsuperuser(ctx):
     print("\nCreated superuser `admin` with password `admin`.")
 
 
-def initialize_database(ctx):
+def initialize_database(ctx, slow=False):
+    """
+    Initialize the database.
+
+    To stop all containers after each management command, pass `slow=True`.
+    """
     print("* Applying database migrations.")
     migrate(ctx)
+    if slow:
```

Comment:
I feel like we should be able to wrap this if statement into a function so we do not have to copy and paste this every few lines

Reply:
It also feels like this should not be necessary in so many places: each operation runs sequentially, so a better solution would be to first make sure the containers are stopped all the way at the top of new-db, after which things should work fine. (the copy-db script does that in Node code, but should be trivial to port to python)

Reply:
I have tried it only at the top with a simple `docker compose down` but that does not solve the issue. It is also not reproducible at which step it happens. 

I have tried a bunch of solutions that would have been more elegant solutions, but none of them worked as I had hoped.

Reply:
@fessehaye I think I have found a solution that is a bit more elegant.